From 54b6da1bafa63d78c0580a3a49f800428ccc6b10 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 20 May 2022 08:29:45 +0300
Subject: [PATCH 43/50] fdts: s32cc: Correct PMIC simple_bus_reg warnings

This corrects dtc warnings:
Warning (i2c_bus_reg): /soc/i2c@402dc000/pf5020_a: I2C bus unit address
format error, expected "9"
Warning (i2c_bus_reg): /soc/i2c@402dc000/pf5020_b: I2C bus unit address
format error, expected "8"
Warning (i2c_bus_reg): /soc/i2c@402dc000/fs5600: I2C bus unit address
format error, expected "18"

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32gxxxa-rdb.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/fdts/s32gxxxa-rdb.dtsi b/fdts/s32gxxxa-rdb.dtsi
index fb06284b5..4e6abd46e 100644
--- a/fdts/s32gxxxa-rdb.dtsi
+++ b/fdts/s32gxxxa-rdb.dtsi
@@ -85,19 +85,19 @@
 		status = "okay";
 	};
 
-	pf5020_a {
+	pf5020_a@9 {
 		compatible = "nxp,pf5020";
 		reg = <0x9>;
 		status = "okay";
 	};
 
-	pf5020_b {
+	pf5020_b@8 {
 		compatible = "nxp,pf5020";
 		reg = <0x8>;
 		status = "okay";
 	};
 
-	fs5600 {
+	fs5600@18 {
 		compatible = "nxp,fs5600";
 		reg = <0x18>;
 		status = "okay";
-- 
2.17.1

